### ISSUE TITLE
SC2102 false positive

### ISSUE THREAD

**Issue by 2tata** [koalaman/shellcheck#2242]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2102
- My shellcheck version (`shellcheck --version` or "online"): version: 0.7.2
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash -e
streamer -s [640x480] -o $ROOT/head.jpeg

```

#### Here's what shellcheck currently says:

streamer -s [640x480] -o "$ROOT"/head.jpeg
            ^-------^ SC2102: Ranges can only match single chars (mentioned due to duplicates).

**Comment by KlzXS** [comment]
Put quotes around it (`"[640x480]"`) so that it properly interprets it as an argument and not a range.

**Comment by koalaman** [comment]
It's a bit of a false lead but definitely a true error: it is indeed interpreted as a range, and the script will break if you have a file named `x` or `0` in the current directory. Make sure to quote it so that it'll be used literally.
